src/routes/gaiaClimaRouter.js

```diff
@@ -1,32 +1,19 @@
-const http = require('http');
+const axios = require('axios');
 
 module.exports = {
```

Comment:
O router é para um microsserviço. Só pode existir router para esporte e ciclone, então isso precisa ser modificado.

Reply:
gaiaNotifyRouter não faz parte de Esporte ou Ciclone, faz parte de Gateway.
No caso ela recebe POST do Gaia-Esporte e envia notificação para o usuário.
Mantém como router separada?

Reply:
Sim, o que for do Gateway pode ter seu próprio router